src/interp/interp.cc

```diff
@@ -41,8 +41,9 @@ const char* GetName(ExternKind kind) {
 
 const char* GetName(ObjectKind kind) {
   static const char* kNames[] = {
-      "Null",   "Foreign", "Trap", "DefinedFunc", "HostFunc", "Table",
-      "Memory", "Global",  "Tag",  "Module",      "Instance", "Thread",
+#define WABT_INTERP_OBJECT_KIND_AS_STRING(Kind) #Kind,
+      WABT_INTERP_OBJECT_KIND(WABT_INTERP_OBJECT_KIND_AS_STRING)
+#undef WABT_INTERP_OBJECT_KIND_AS_STRING
```

Comment:
I'm not sure this pattern is really worth the added obfuscation that comes with it.

Elsewhere in the codebase we do this instead, which is probably good enough here too?:

```WABT_STATIC_ASSERT(WABT_ARRAY_SIZE(g_reloc_type_name) == kRelocTypeCount);``

Reply:
Good idea. I have reworked the check to follow the other enums in the code.